src/test/java/app/component/Operator.java

```diff
@@ -52,12 +63,20 @@ public String toString() {
     private record V2Envelope(String envelope, byte[] nonce) {
     }
 
+    private static final SecureRandom SECURE_RANDOM = new SecureRandom();
     private static final String CLIENT_API_KEY = EnvUtil.getEnv("UID2_E2E_API_KEY");
     private static final String CLIENT_API_SECRET = EnvUtil.getEnv("UID2_E2E_API_SECRET");
     private static final String CLIENT_API_KEY_BEFORE_OPTOUT_CUTOFF = EnvUtil.getEnv("UID2_E2E_API_KEY_OLD");
     private static final String CLIENT_API_SECRET_BEFORE_OPTOUT_CUTOFF = EnvUtil.getEnv("UID2_E2E_API_SECRET_OLD");
+    private static final String CLIENT_SIDE_TOKEN_GENERATE_SUBSCRIPTION_ID = EnvUtil.getEnv("UID2_E2E_SUBSCRIPTION_ID");
+    private static final String CLIENT_SIDE_TOKEN_GENERATE_SERVER_PUBLIC_KEY = EnvUtil.getEnv("UID2_E2E_SERVER_PUBLIC_KEY");
+    private static final String CLIENT_SIDE_TOKEN_GENERATE_ORIGIN = EnvUtil.getEnv("UID2_E2E_ORIGIN");
```

Comment:
Do both valid and invalid origins have to be passed in dynamically? Can they be hardcoded instead?